python/interpret_community/dataset/dataset_wrapper.py

```diff
@@ -218,6 +238,21 @@ def summary_dataset(self):
         """
         return self._summary_dataset
 
+    def _set_default_index_cols(self, dataset):
+        if dataset.index.name is not None:
+            self._default_index_cols = dataset.index.name
+        if dataset.index.names is not None:
+            self._default_index_cols = dataset.index.names
+
+    def set_index(self):
```

Comment:
Are we expecting users to use this directly?

Reply:
I guess we don't even really expect them to use dataset wrapper directly yet.  But if they are using dataset wrapper, it seems like a good thing to expose since reset_index is also public, and both exist as public functions on pandas dataframes.